AppController/djinn.rb

```diff
@@ -539,6 +539,11 @@ class Djinn
     'use_spot_instances' => [ TrueClass, nil, false ],
     'user_commands' => [ String, nil, true ],
     'verbose' => [ TrueClass, 'False', true ],
+    'write_nodes_stats_log' => [ TrueClass, 'False', true ],
+    'write_processes_stats_log' => [ TrueClass, 'False', true ],
+    'write_proxies_stats_log' => [ TrueClass, 'False', true ],
+    'write_detailed_processes_stats_log' => [ TrueClass, 'False', true ],
+    'write_detailed_proxies_stats_log' => [ TrueClass, 'False', true ],
```

Comment:
There is a comment just before the parameters hash:
```
  # List of parameters allowed in the set_parameter (and in AppScalefile
  # at this time). If a default value is specified, it will be used if the
  # parameter is unspecified. The last value (a boolean) indicates if the
  # parameter's value is of a sensitive nature and shouldn't be printed in
  # the logs.
```
So the last value indicates if controller need to replace value of a parameter with `'*****'`.